manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate
+
+PackageIdentifier: SilverlakeSoftwareLLC.Velocity

```

```suggestion
PackageIdentifier: SilverlakeSoftware.Velocity
```
You'll also need to change the folder name and its files from `SilverlakeSoftwareLLC` to `SilverlakeSoftware` using GitHub's desktop client on Windows.